crates/tauri-cli/src/interface/rust/desktop.rs

```diff
@@ -440,7 +440,7 @@ mod terminal {
       // INVALID_HANDLE_VALUE. Use an alternate method which works
       // in that case as well.
       let h = CreateFileA(
-        "CONOUT$\0".as_ptr() as PCSTR,
+        c"CONOUT$".as_ptr() as PCSTR,
```

Comment:
TIL

Reply:
Same, almost thought clippy is trolling me